docs/exercises/3-encryption_sdk.rst

```diff
@@ -236,14 +236,14 @@ we won't need to keep around the key ID, so we can discard that value.
     .. group-tab:: Java
 
         We won't need the class attribute for ``keyID``, so replace that with ``masterKey``
-        for the KMS master key.
+        for the KMS Master Key.
 
         .. code-block:: java
            :lineno-start: 56
 
             private final KmsMasterKey masterKey;
 
-        In our constructor, we'll create the master key like so:
+        In our constructor, we'll create the Master Key like so:
```

Comment:
Except where we initially present the concepts, we do not treat master key, data key, or crypto materials manager as proper nouns.

Reply:
This change was to treat them as proper nouns in this workshop, to help distinguish the SDK-specific concepts from the concepts in general.

I'm not sure whether or not this is the right general solution, but it seemed helpful in the context of attempting to help people grok KMS and Encryption SDK specifics in a short time window.